aflr/api_resources/file.py

```diff
@@ -1,19 +1,46 @@
+from posixpath import basename
 import aflr
 from aflr.api_request import APIRequest
+import os
 
 
 class File(APIRequest):
+    custom_audio_resource_path = "/file/customaudio/"
     OBJECT_NAME = "file"
+    resource_path = "/file"
 
-    def __init__(self):
-        super().__init__()
-        self.url = self.api_base + "/file"
+    @classmethod
```

Comment:
move this method to helper_classes. see my last comment.

Reply:
done